### PR TITLE
DTGB-144: Added SassDoc generation.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/public/sassdocs/assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* global document */
+
+(function ($, global) {
```
Comment (BartDelrue) [inline styleguide/public/sassdocs/assets/js/main.js]: No Jquery please.
I don't see it in de sassdoc dependencies, nor in our own package.json.

File: styleguide/gulpfile.js [hunk 2]
```diff
@@ -660,6 +665,10 @@ gulp.task('iconfont', () => {
     .pipe(gulp.dest('./public/styleguide/fonts/'));
 });
 
+gulp.task('sassdoc', () => {
```
Comment (BartDelrue) [inline styleguide/gulpfile.js]: This does nothing.

File: styleguide/gulpfile.js [hunk 3]
```diff
@@ -198,6 +199,10 @@ gulp.task('styles:inject', () => {
 gulp.task('styles:dist', (callback) => {
   _sassLint(false)
     .pipe(sourcemaps.init())
+    .pipe(sassdoc({
```
Comment (BartDelrue) [inline styleguide/gulpfile.js]: I believe this should be in build.

As this gulpfile is currently being refactored, please create a seperate task for sassdoc and indicate where it should be used.

**Review by BartDelrue — CHANGES_REQUESTED** [review]

File: styleguide/gulpfile.js (hunk 3 above)
Comment (meirege) [reply styleguide/gulpfile.js]: Changed it to a separate task. I'll indicate in the PR for the gulp file refactoring where it should be used.

File: styleguide/gulpfile.js (hunk 2 above)
Comment (meirege) [reply styleguide/gulpfile.js]: Yup, you are right.

**Review by delrueba — APPROVED** [review]

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**